rust/frontend/src/impls/service_based_frontend.rs

```diff
@@ -623,6 +624,14 @@ impl ServiceBasedFrontend {
             database: database_name,
             collection_id: source_collection_id.0,
             latest_collection_logical_size_bytes,
+            request_execution_time_ns: Some(
```

Comment:
One potential option is for service based frontend to accept a partially built MeterEvent, and add the subsequent data to it and submit it.